src/extensions.rs

```diff
@@ -1,11 +1,36 @@
 use std::any::{Any, TypeId};
 use std::collections::HashMap;
+use std::hash::{BuildHasherDefault, Hasher};
 use std::fmt;
 use std::hash::BuildHasherDefault;
 
-use fnv::FnvHasher;
+pub(crate) struct IdHasher {
```

Comment:
Do we actually use `IdHasher` outside of this module? It seems unnecessary to put `pub(crate)` here